transaction-scenarios/src/scenarios/fungible_resource.rs

```diff
@@ -0,0 +1,435 @@
+use crate::internal_prelude::*;
+use radix_engine::types::*;
+use radix_engine_interface::api::node_modules::ModuleConfig;
+use radix_engine_interface::blueprints::account::ACCOUNT_TRY_DEPOSIT_OR_ABORT_IDENT;
+use radix_engine_interface::*;
+
+pub struct FungibleResourceScenarioConfig {
+    pub user_account_1: VirtualAccount,
+    pub user_account_2: VirtualAccount,
+}
+
+#[derive(Default)]
+pub struct FungibleResourceScenarioState {
+    pub max_divisibility_fungible_resource: Option<ResourceAddress>,
```

Comment:
The `State` wrapper was designed to be slightly more intuitive to get/set, but using `Option` is totally fine too